docs/_data/portland-2024-config.yaml

```diff
@@ -155,11 +155,11 @@ about:
   job_fair_room: Sponsor area
 
 cfp:
-  url: https://pretalx.com/wtd-portland-2024/cfp  # Open November 6 (Mon) -> 9 weeks to submit
-  ends: "January 15, 2024"                        # (Mon) 1 week, to review. It'll take a little longer to announce
-  notification: "January 30, 2024"                # (Mon) 8 weeks to write and prepare
-  slides_by: "April 7, 2024"                      # (Mon) 1 week before
-  preview: "TODO"                                 # Preview URL TODO
+  url: https://pretalx.com/wtd-portland-2024/cfp # Open November 6 (Mon) -> 9 weeks to submit
+  ends: "January 15, 2024" # (Mon) 1 week, to review. It'll take a little longer to announce
+  notification: "January 30, 2024" # (Mon) 8 weeks to write and prepare
+  slides_by: "April 7, 2024" # (Mon) 1 week before
+  preview: "TODO" # Preview URL TODO
```

Comment:
I feel like this is not an improvement ;-)

Reply:
Ah yea.. didn't catch that it removed the spacing that was intentional... Really wish it standardized the quotes and whatnot, would be nice to have a good YAML linter.

Reply:
Aye.

I guess we could also look at 

```
comment
comment 
comment

info 
info 
info
```